packages/pn-commons/src/utility/fetch.configuration.utility.ts

```diff
@@ -5,7 +5,7 @@
 
 export async function fetchConfiguration(): Promise<any> {
   try {
-    const res = await fetch('conf/config.json', {
+    const res = await fetch(import.meta.env.BASE_URL + 'conf/config.json', {
```

Comment:
Because of a problem regarding the access of a SEND product using a relative path. An usual case is when the user opens a document attached to the detail of a notification, and then clicks the "Back" browser button to get back to the SEND app (since now the attached document is opened in the same tab as the app). 
In that case, the app as built through Vite considers `conf/config.json` to be a path relative to the path used to open the app, hence it attempts to read e.g.
```https://cittadini.dev.notifichedigitali.it/notifiche/YAVD-MEGW-VMWL-202310-N-1/conf/config.json```

The way I found to make this file location absolute, according to https://vitejs.dev/guide/build.html#public-base-path, was to prepend `import.meta.env.BASE_URL`. The Vite build process guarantees that `import.meta.env` is actually injected on app launch.

IMHO this change should remain.